chunks/tool-bazel/Dockerfile

```
@@ -0,0 +1,11 @@
+ARG base
+FROM ${base}
+
+USER gitpod
+
+# Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
+ENV TRIGGER_REBUILD=1
+
+# store the blaze/bazel cache in a place where it will be persisted
+# See also https://docs.bazel.build/versions/main/output_directories.html
+RUN mkdir -p /home/gitpod/.cache/bazel && ln -s /home/gitpod/.cache/bazel /workspace/.bazel
```

This change will be not be effective as this `/workspace` dir will be overlaid via a mount once a workspace session will be started by Gitpod.

It needs to be done from something like `.bashrc` with a conditional check for `GITPOD_REPO_ROOT`'s existence.
```suggestion
RUN RUN ts="$HOME/.bashrc.d/700-bazel" && printf '%s\n' \
						't="$HOME/.cache/bazel"' \
						's="${t//$HOME/\/workspace}"' \
						'test ! -L "$t" && rm -rf "$t" && mkdir -p "$s" && ln -s "$s" "$t"' \
						"rm -f $ts" > "$ts"
```

While there is an env var which we could set in such way too via bash as it's stated here: https://bazel.build/docs/output_directories#layout

But then I came across [this](https://stackoverflow.com/questions/40775920/how-can-i-make-bazel-use-external-storage-when-building#comment98156737_40775921) comment which got me thinking, so I guess we should still continue with the symlink method unless that comment is proven otherwise (I'm not familiar with `bazel`, so no idea)